identity_credential/src/domain_linkage/domain_linkage_configuration.rs

```diff
@@ -128,6 +128,9 @@ mod __fetch_configuration {
       if domain.scheme() != "https" {
         return Err(DomainLinkageError("domain` does not use `https` protocol".into()));
       }
+      if domain.path() != "/" || domain.query().is_some() || domain.fragment().is_some() {
```

Comment:
Not sure about the part `domain.path() != "/"` would it return false if the URL was `http://www.example.com/` and not `http://www.example.com` ?  we can probably also add positive tests for these cases too.